scheds/rust/scx_layered/src/bpf/util.bpf.c

```diff
@@ -54,15 +54,21 @@ static char *format_cgrp_path(struct cgroup *cgrp)
 	return path;
 }
 
-static inline bool match_prefix(const char *prefix, const char *str, u32 max_len)
+int match_prefix(const char *prefix, const char *str, u32 max_len)
```

Comment:
I hit verifier kernel panics when trying to use a [bool](https://github.com/sched-ext/scx/compare/main...hodgesds:scx:kernel-panic#diff-032d077d10c532e766a8caaa9a0eee678ceedf5c5ee6e153e3fed10aa2fede4aR57).